### PR TITLE
;doubleornothing

### PR DESCRIPTION
**Pull request by thexpiredpear**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -970,7 +970,52 @@ async def lmao(ctx):
...
+        embed.set_author(name="Gambling")
+        embed.add_field(
+            name="Notice",
+            value="`Please specify how many points you would like to gamble`",
+            inline=True,
+        )
+
+    else:
+        if points <= userpoints:
+            if random.randint(0, 1) == 1:
```
Comment (gubareve) [inline bot.py:987]: should be
`if random.randint(0,9) in [0,1,2,3]`
we cant make gambling too good

File: bot.py [hunk 2]
```diff
@@ -970,7 +970,52 @@ async def lmao(ctx):
...
+                embed = discord.Embed(color=discord.Colour.from_rgb(r, g, b))
+                embed.set_image(url="https://cdn.discordapp.com/attachments/716471303682523147/724374290786549840/coinflip.gif")
+                embed.set_author(name="Gambling")
+                embed.add_field(
+                    name="You won!",
+                    value="`You have won {} points!`".format(points),
```
Comment (gubareve) [inline bot.py:993]: add a poggers cause why not

File: bot.py [hunk 3]
```diff
@@ -970,7 +970,52 @@ async def lmao(ctx):
     else:
         await ctx.send("Buy this gif in the shop!")
 
+@client.command()
+async def doubleornothing(ctx, points=None):
```
Comment (gubareve) [inline bot.py:974]: add a alias for ;gamble

Comment (thexpiredpear) [reply bot.py:974]: there might be a pot later

**Pull request merged**